docker-compose.yml

```diff
@@ -37,6 +37,7 @@ services:
     environment:
       FLASK_APP: api.py
       FLASK_RUN_PORT: 5434
+      OAUTHLIB_INSECURE_TRANSPORT: 1
```

Comment:
Can we somehow prevent this from being `1` on production?

Reply:
The library requests_oauthlib triggers an InsecureTransportation error without this environment variable. My understanding is that Nginx encrypts communication from clients to the Flask API so I'm not sure where this error comes from. I'm wondering if it's really a problem or not and I don't know how to fix it at the moment.

Reply:
Ok, so I would just leave it like that. Mabye it could also be, because the communication between Flask (requests_oauthlib) and nginx is HTTP, while the connection between nginx and the client is HTTPS.